openfisca_country_template/tests/reforms/modify_social_security_brackets.yaml

```diff
@@ -0,0 +1,30 @@
+# Test files describe situations and their expected outcomes
+# We can run this test on our command line using `openfisca-run-test social_security_contribution.yaml`
+
+# This test is a partial adaptation of `social_security_contribution.yaml` to match the modifications introduced by the reform of the same name.
+
+# Note the `reforms: ` key in the below YAML blocks.
+
+- name: Person earning 2000 in 2013-01
```

Comment:
The date doesn't match :wink:

But I guess that shows the name is bad if it is just a copy of data that is readily available below. What about something like “No social security contribution on small salaries”?

(repeat below)